pkg/api/types.go

```diff
@@ -19,10 +19,6 @@ import (
 	v20170831 "github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20170831"
 	v20180331 "github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20180331"
 	"github.com/Azure/aks-engine/pkg/api/common"
-	v20160330 "github.com/Azure/aks-engine/pkg/api/v20160330"
```

Comment:
Are we keeping `v20170831` and `v20180331` around in the AKS Engine codebase?

Reply:
at the time of this PR, I had not removed the usage from the RP.
I'll send a 2nd PR to get rid of agentpoolonly apis